core/spec/models/spree/shipment_spec.rb

```diff
@@ -1,8 +1,8 @@
 require 'spec_helper'
 require 'benchmark'
 
-describe Spree::Shipment, :type => :model do
-  let(:stock_location) { create(:stock_location) }
+describe Spree::Shipment, type: :model do
+  subject { shipment }
```

Comment:
for what it's worth i think keeping shipment is better because it's more explicit